### PR TITLE
Advanced harness suite test

### PR DESCRIPTION
**Pull request by yodzhubeiskyi**

### CODE REVIEW

**Review by PavloTytarchuk — APPROVED** [review]
Looks good to me, @yodzhubeiskyi, nice job.

File: pom.xml [hunk 1]
```diff
@@ -58,7 +58,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <liquibase.version>0-SNAPSHOT</liquibase.version>
+        <liquibase.version>4.18.0</liquibase.version>
```
Comment (kristyldatical) [inline pom.xml:61]: Should we not go to the latest version of Liquibase which was released last week i.e. `4.19.0`?

Comment (kristyldatical) [reply pom.xml]: @KushnirykOleh -- could you check on this too please.

**Review by KushnirykOleh — APPROVED** [review]

**Review by kristyldatical — APPROVED** [review]
Nice work @yodzhubeiskyi, thank you!

**Pull request merged**